r/R/dataset.R

```diff
@@ -228,6 +228,86 @@ open_dataset <- function(sources,
   )
 }
 
+#' Open a multi-file dataset of CSVs
+#'
+#' A wrapper around [open_dataset] which explicitly includes parameters specific to reading CSV files
+#'
+#' @inheritParams open_dataset
+#' @inheritParams read_csv_arrow
+#'
```

Comment:
It might be helpful to include an example here that shows why you would want this function, i.e. what it simplifies. (A reprex of that would also be nice to include on the PR description to demonstrate why this needs to exist.)

Other things worth considering:

* Working this function into a vignette
* xrefs from the `open_dataset` docs (`@seealso`; an inline reference in the "format" argument; etc.) and the `read_delim_arrow` docs.